FirebaseAuth/Sources/AuthProvider/OAuth/FIROAuthCredential_Internal.h

```diff
@@ -57,6 +57,14 @@ NS_ASSUME_NONNULL_BEGIN
                             secret:(nullable NSString *)secret
                       pendingToken:(nullable NSString *)pendingToken NS_DESIGNATED_INITIALIZER;
 
+- (instancetype)initWithProviderID:(NSString *)providerID
+                           IDToken:(nullable NSString *)IDToken
+                          rawNonce:(nullable NSString *)rawNonce
+                       accessToken:(nullable NSString *)accessToken
+                            secret:(nullable NSString *)secret
+                       displayName:(nullable NSString *)displayName
+                      pendingToken:(nullable NSString *)pendingToken NS_DESIGNATED_INITIALIZER;
+
```

Comment:
Instead of defining a new init method, can you just add a `displayName` parameter to the existing method above, and then update the existing callers of that method to pass in a `nil` `displayName`?